src/main/java/com/entropyteam/entropay/employees/services/ContractService.java

```diff
@@ -153,5 +152,17 @@ public Contract checkActiveContract(Contract contractToCheck) {
         }
         return contractToCheck;
     }
+    @Override
+    public Map<String, Object> getRestrictedFields(AppRole userRole){
+        Map<String,Object> restrictedFields = new HashMap<>();
+                if(AppRole.ROLE_MANAGER_HR.equals(userRole)){
+                    
+                    Optional<Role> role = roleRepository.findByDeletedIsFalseAndName(CONTRACT_ROLE_HHRR);
```

Comment:
Let's change this method to find in the table role:  
`select * from role where delete = false and name like '%HR%'`